### DISCUSSION TITLE
Visualizing placement axes

### DISCUSSION THREAD

**Discussion by RickBrice** [IfcOpenShell/IfcOpenShell#4472] [Q&A]
The result of evaluating an IfcLinearPlacement.RelativePlacement is a 4x4 matrix. It's pretty easy to plot X,Y,Z for the point. I'm looking for a way to graphically display the X, Y, Z vectors of the placement coordinate system.

Any ideas?

**Comment by RickBrice** [comment]
I found a method.
https://stackoverflow.com/questions/27023068/plotting-3d-vectors

**Comment by aothms** [comment]
In Blender you have also the `Empty > Plain axis` object. Like any other kind of object you can read/assign to `matrix_world`.

![afbeelding](https://github.com/IfcOpenShell/IfcOpenShell/assets/1096535/f7711925-0c96-4937-8a58-f2a73b22af2f)
